src/components/basic/text/Text.tsx

```diff
@@ -11,7 +11,8 @@ import styled, { css, DefaultTheme } from 'styled-components';
 import { getColor } from '../../../theme/theme-utils';
 import { AnyColor } from '../../../types/utils';
 
-type TextOverflow = 'ellipsis' | 'break-word';
+export const textOverflowValues = ['ellipsis', 'break-word'] as const;
+type TextOverflow = (typeof textOverflowValues)[number];
```

Comment:
Just use the 2 possible value directly in the type and Storybook is then able to show the radio on its own

Reply:
@CataldoMazzilli I meant that we can remove the helper type TextOverflow, and set the overflow type prop directly like this
```ts
TextProps ... {
  ...
 overflow: 'break-word' | 'ellipsis';
  ...
}
```
then you can remove the docs/utils argtype for the overflow, since react-docgen-typescript is then able to recognize the simple union, and shows the radio automatically